### ISSUE TITLE
Handle multiple entry point

### ISSUE THREAD

**Issue by lionelB** [Yomguithereal/kotatsu#98]
Hello,

It seems that kotatsu doesn't support multiple entry point.
also I would be cool to config the output folder for build assets (default seems to be /build)

**Comment by Yomguithereal** [comment]
It does support multiple entry points when using a config. This should be fixed when the issue with ES6 config is cleared.

**Comment by lionelB** [comment]
I guess the problem come from 
https://github.com/Yomguithereal/kotatsu/blob/master/src/createCompiler.js#L105

**Comment by lionelB** [comment]
even if output is added, `kotatsu serve` always create a bundle in `/build/bundle.js`

**Comment by Yomguithereal** [comment]
Any news @lionelB?

**Comment by Yomguithereal** [comment]
Sorry, I misclosed.

**Comment by lionelB** [comment]
will check asap

**Comment by bebraw** [comment]
Has this been fixed?

I have configuration like this:

**webpack.config.js**

```javascript
const path = require('path');

module.exports = {
  entry: {
    style: path.join(__dirname, 'app/main.css')
  }
};
```

**package.json**

```json
...
"start": "kotatsu serve --babel app/index.jsx --config ./webpack.config.js"
...
```

The idea is that kotatsu should inject the additional entry to the output. I inject CSS this way instead of using a `require` as then my caching approach works correctly (separate chunks -> proper invalidation).

**Comment by bebraw** [comment]
I patched it up with a little check like this at **createCompiler.js**

```javascript
...

  if(opts.config.entry) {
    entryConfig = entryConfig.concat(opts.config.entry);
  }

  // Building the output
  var outputConfig = _.merge(
...
```

Obviously that's really brittle and handles just array input (no support for object format).

**Comment by Yomguithereal** [comment]
The current logic is that the given file (by CLI) will take precedence over the configuration. The logic you'd want is for both to be taken into account? Is that the case?

**Comment by bebraw** [comment]
@Yomguithereal Yeah, that would be the idea. I pushed all the entries to webpack configuration based on that. I suppose at minimum kotatsu should give a warning if it overrides some configuration like this. Easier to see what's going on. I'm not sure what's the best logic here.
